script/utils.js

```diff
@@ -12,7 +12,7 @@ const runCommand = (cmd, forcedExitCode = null) => {
   });
 };
 
-const runCommandSync = (cmd, forcedExitCode = null) => {
+const runCommandSync = cmd => {
```

Comment:
A neat trick I learned from someone during code review is you can also prefix with an underscore to have the linter ignore it but retain the variable - `_forcedExitCode`

Reply:
Given that nothing happens with that argument I think it makes sense to omit it? But it kinda depends on what they were planning on doing with this I think 🙂

Reply:
I did not know that btw, very handy!! 🔥

Reply:
I think I learned it from @bradl3yC awhile ago!